All Nx cli commands fail on 13.9

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
All NX cli commands fail for me with no output, even `--help`.
<!-- What is the behavior that currently you experience? -->

## Expected Behavior
Nx cli commands should work - at least `--help` should. 

This started to happen since yesterday, giving me the hint that this might be a regression in `13.9`
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->

## Steps to Reproduce
I already had a monorepo - so I ran this command according to docs: 
`npx add-nx-to-monorepo`

I've done this multiple times, after switching to a clean branch and clearing `node_modules`, and running `npm install`. 

When I pin the `@nrwl/*` versions to `13.8`, it works fine.

<!-- Help us help you by making it easy for us to reproduce your issue! -->

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->

### Failure Logs
Nx commands return nothing except a non-zero return code. Stdout / stderr is completely empty, even for `npx nx`
<!-- Please include any relevant log snippets or files here. -->

### Environment
Node : 14.17.5
OS   : darwin x64
npm  : 6.14.14
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->

The following version gives me this error:
```
"@nrwl/cli": "latest",
"@nrwl/tao": "latest",
"@nrwl/workspace": "latest",
```

Although the actual issue you're reporting might not be fixed in `13.9.1`, as I'm now getting:
```
Unexpected error:
Error: Cannot find module 'nx/src/cli/run-cli.js'
```

--- a/packages/nx/package.json
+++ b/packages/nx/package.json
@@ -83,6 +83,7 @@
       "@nrwl/web",
       "@nrwl/js",
       "@nrwl/cli",
+      "@nrwl/tao",
       {
         "package": "@nrwl/nx-cloud",
         "version": "latest"

--- a/packages/workspace/package.json
+++ b/packages/workspace/package.json
@@ -48,6 +48,7 @@
       "@nrwl/web": "*",
       "@nrwl/js": "*",
       "@nrwl/cli": "*",
+      "@nrwl/tao": "*",
       "@nrwl/nx-cloud": "latest",
       "@nrwl/react-native": "*",
       "@nrwl/detox": "*",

